### PR TITLE
Migrate to use dp-renderer

### PR DESCRIPTION
**Pull request by rav-pradhan**
### What

Migrate from `dp-frontend-renderer` to `dp-renderer`

Currently, the version of `dp-renderer` is set to a specific commit hash [as the work to implement the rendering library logic is still in review](https://github.com/ONSdigital/dp-renderer/pull/4). I will update this app with a tagged version once I have updated `dp-renderer`

### How to review

- Sense check the code and changes
- Check that tests pass and you can view the cookies preference page locally without `dp-frontend-renderer` running

You will require at least the [web journey](https://github.com/ONSdigital/dp/blob/main/guides/INSTALLING.md#web-journey) running locally. 

Once that is running, you should be able to go to `localhost:20000/cookies` and see the cookies preference page. It should look exactly like what you would find on the [live site](https://www.ons.gov.uk/cookies) - but your local environment should not be using `dp-frontend-renderer` at all

### Who can review

Anyone

### CODE REVIEW

**Review by redhug1 — COMMENTED** [review]
In main.go goland reports line 89 as unreachable and it truly is unreachable, which makes the comments on lines 87 and 88 totally misleading and in error. I think those comment lines 87 and 88 should be removed and the return nil on line 89 removed or re-coded to return the value that line 84 is returning or some such like.

File: handlers/handlers.go [hunk 1]
```diff
@@ -137,22 +121,16 @@ func edit(w http.ResponseWriter, req *http.Request, rendC RenderClient, siteDoma
 	cookies.SetPreferenceIsSet(w, siteDomain)
 	cookies.SetPolicy(w, cp, siteDomain)
 	isUpdated := true
-	err = getCookiePreferencePage(w, req, rendC, cp, isUpdated, lang)
+	getCookiePreferencePage(w, req, rendC, cp, isUpdated, lang)
 	if err != nil {
```
Comment (redhug1) [inline handlers/handlers.go:125]: Where is this 'err' value being populated to be being checked here ?

**Review by redhug1 — CHANGES_REQUESTED** [review]
See other comment

**Review by redhug1 — COMMENTED** [review]
Here's a list of other minor issues that could be attended to:
routes.go
line 17, unused parameter ‘cfg’

cookie-preferences.go:
line 5, Comment should have the following format ‘CookiesPreference ...’ (with an optional leading article) … maybe ?

mapper_test.go
line 9, redundant alias

handlers_test.go
line 115, Expression ‘hasBeenUpdated’ is always ‘true’
line 115, Expression ‘cookiesPreferenceIsSet’ is always ‘true’
line 249, Unused parameter ‘isUpdated’

handlers.go
line 124, Expression ‘isUpdated’ is always ‘true’
line 125, Condition is always ‘false’ because ‘err’ is always ‘nil’
line 135, Expression ‘isUpdated’ is always ‘false’
line 6, Redundant alias

File: handlers/handlers.go (hunk 1 above)
Comment (rav-pradhan) [reply handlers/handlers.go:125]: Yep, oversight from me when updating the code here. This `err` check is not needed - will remove

**Review by redhug1 — APPROVED** [review]

**Pull request merged**